django_bulk_load/bulk_load.py

```diff
@@ -219,6 +220,8 @@ def bulk_update_models(
     list is changed) (i.e. update_on/last_modified)
     :param update_if_null_field_names: Fields that only get updated if the new value is NULL or existing
     value in the DB is NULL.
+    :param update_where: Function that returns a Composable that is used to filter the update query. Cannot be used
+    with model_changed_field_names or update_if_null_field_names
```

Comment:
I updated (locally) the test you added for `update_where` by adding the following to the call to `bulk_update_models`:

- `update_if_null_field_names=["string_field"]`

And this causes the test to fail (this seems expected). But I added many variations of `model_changed_field_names` and things seem to work as intended in the simple test case. Just flagging that it might make sense to clarify this docstring, or if we're worried about unexpected behavior, should we sanitize the inputs to `bulk_update_models` and `bulk_upsert_models` to check that callers aren't using `update_where` and these alternatives and either warn or (my preference) fail to run the load operation.

```suggestion
    :param update_where: Function that returns a Composable that is used to filter the update query. Should not be used
    with model_changed_field_names or update_if_null_field_names (can lead to unexpected behavior)
```